### PR TITLE
Use default mirror in docker build

### PR DESCRIPTION
**Pull request by Yagoor**
As we can see on #1845, the mirror://mirrors.ubuntu.com/mirrors.txt is not working.
I tried locally and it also does not work.
@pjonsson I can see that you added it. Is it okay to revert it?

### CODE REVIEW

File: tools/docker/Dockerfile [hunk 1]
```diff
@@ -7,9 +7,10 @@ USER root
 
 # Tools
 RUN apt-get -qq update && \
-    apt-get -qq -y --no-install-recommends install ca-certificates > /dev/null && \
-    apt-get -qq update && \
-    apt-get -qq -y --no-install-recommends install gnupg software-properties-common > /dev/null
+    apt-get -qq -y --no-install-recommends install \
+      gnupg \
```
Comment (pjonsson) [inline tools/docker/Dockerfile:11]: While you're touching this, could you sort these packages alphabetically so they are consistent with the other large block of packages?

**Comment by pjonsson** [comment]
The mirrors work wonderfully in Sweden, but I can't argue against the CI failures.

File: tools/docker/Dockerfile (hunk 1 above)
Comment (Yagoor) [reply tools/docker/Dockerfile:12]: Done. :)

**Comment by Yagoor** [comment]
> The mirrors work wonderfully in Sweden, but I can't argue against the CI failures.

This is interesting. I'm just on the other side of Øresund and it does not work locally either.

**Review by pjonsson — APPROVED** [review]

**Comment by pjonsson** [comment]
> > The mirrors work wonderfully in Sweden, but I can't argue against the CI failures.
> 
> This is interesting. I'm just on the other side of Øresund and it does not work locally either.

Even more strange, I get su.se, umu.se, c0urier.net, and a fourth mirror that I can't remember the name of, and they are all fast and stable.

**Review by nfi — APPROVED** [review]

**Pull request merged**